assets/sass/components/dashboard/_googlesitekit-DashboardPageSpeed.scss

```diff
@@ -37,21 +37,41 @@
 		}
 	}
 
+	.googlesitekit-pagespeed-widget__content-wrapper {
+		position: relative;
+	}
+
+	.googlesitekit-pagespeed-widget__content-wrapper--loading {
+		.googlesitekit-pagespeed-widget__content {
+			visibility: hidden;
+		}
+	}
+
+	.googlesitekit-pagespeed-widget__progress-bar {
+		left: 50%;
+		position: absolute;
+		top: 50%;
+		transform: translate(-50%, -50%);
+		width: 100%;
+	}
+
 	.googlesitekit-pagespeed-widget__header {
-		align-items: flex-end;
```

Comment:
The tweaks in `.googlesitekit-pagespeed-widget__header` and `.googlesitekit-pagespeed-widget__data-src-tabs`  are needed to accommodate the presence of the new "How to improve" tab in the widget header.

Reply:
Update: I noticed that these changes had introduced a glitch in the tab bar, so reverted and introduced a simpler modification, see 11db04aba6bb9a70b900933276e3f4a95debb9c3.